zellij-server/src/panes/terminal_pane.rs

```diff
@@ -387,7 +387,6 @@ impl TerminalPane {
     pub fn read_buffer_as_lines(&self) -> Vec<Vec<TerminalCharacter>> {
         self.grid.as_character_lines()
     }
-    #[cfg(any(feature = "test", test))]
     pub fn cursor_coordinates(&self) -> Option<(usize, usize)> {
```

Comment:
This function is only used in e2e testing, is it okay to have it in normal builds as well `#[cfg(any(feature = "test", test))]`? Maybe add a comment stating that it's required only for e2e tests?

Reply:
Sure. I think it's not a problem to have this in normal builds as well (it's just a few lines of code). I can also imagine occasions where we might need it in the app itself at some point. So maybe we can leave it in and suppress the compiler warning?